drivers/rawexec/driver_unix.go

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+//go:build !windows
+
+package rawexec
+
+import (
+	"fmt"
+	"os/user"
+
+	"github.com/hashicorp/nomad/helper/users"
+	"github.com/hashicorp/nomad/plugins/drivers"
+)
+
+func (d *Driver) Validate(cfg drivers.TaskConfig) error {
```

Comment:
We're not validating anything but the user here, right? Should this function signature be identical to `validateUserIds` in the `exec` driver?

Reply:
Alternatively, we could move the `OOMScoreAdj` check from the `StartTask` function into here?

Reply:
I was trying to keep as much as possible from the original code, but since the windows/unix diasctintion is not necessary anymore, it makes sense to remove this, lets do it.

Reply:
right now it only checks for users, but it leaves a clear space for validations and might help us keep the code organised later